### ISSUE TITLE
ios14 video crashes when app goes to background and reopens --> Software caused connection abort

### ISSUE THREAD

**Issue by enigmablue** [TheWidlarzGroup/react-native-video#2307]
The screen (video) goes black after closing the app and reopening it.
I'm unable to catch any error (nothing is thrown onError).

The only thing i see on console are these messages. My only solution is to track when the app goes to the background and wipe out the video but its quite an ugly solution.

Anyone know what this is about? 

```
[connection] nw_read_request_report [C97] Receive failed with error "Software caused connection abort"
[connection] nw_read_request_report [C86] Receive failed with error "Software caused connection abort"
BackgroundSession <6892063E-158A-4FC1-83AC-1CD21AC5D9FD> connection to background transfer daemon interrupted
BackgroundSession <B24A246C-F710-4E48-B3CF-3A84D17DC1C7> connection to background transfer daemon interrupted
BackgroundSession <6892063E-158A-4FC1-83AC-1CD21AC5D9FD> connection to background transfer daemon invalidated
BackgroundSession <B24A246C-F710-4E48-B3CF-3A84D17DC1C7> connection to background transfer daemon invalidated
```

**Comment by qinjialei24** [comment]
same problem

**Comment by orange-brother** [comment]
+1

**Comment by orange-brother** [comment]
@hueniverse 

is this issue will be fixed in 6.0.0?

**Comment by hueniverse** [comment]
@patissier-boulanger I don't know. Please check and report back.
